### PR TITLE
Bump eslint

### PR DESCRIPTION
**Pull request by int64ago**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines         
- [x] Tests for the changes have been added (for bug fixes / features)          
- [x] Docs have been added / updated (for bug fixes / features)                      

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

update eslint

* **What is the related issue?** (Use `#issue_id` to relate an open issue)

* **Does this PR introduce a breaking change?** 

* **Other information**:

### CODE REVIEW

File: packages/svrx-util/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "cosmiconfig": "^5.2.1",
     "lodash": "^4.17.11",
     "npm": "^6.9.0",
-    "npmi": "^2.0.1",
+    "npmi": "^4.0.0",
```
Comment (xuchaoying) [inline packages/svrx-util/package.json]: 不应该在这个PR里修改， see https://github.com/x-orpheus/svrx/pull/44

File: package.json [hunk 2]
```diff
@@ -19,14 +19,12 @@
     "prepublish": "npm run build"
   },
   "devDependencies": {
-    "@commitlint/cli": "^7.5.2",
+    "@commitlint/cli": "^8.1.0",
```
Comment (xuchaoying) [inline package.json]: 不应该在这个pr里修改

Comment (int64ago) [reply package.json]: > 不应该在这个pr里修改

It's modified by `npm audit fix`, this PR includes three commits.

File: packages/svrx-util/package.json (hunk 1 above)
Comment (int64ago) [reply packages/svrx-util/package.json]: > 不应该在这个PR里修改， see #44

The deps bump PR should be merged in time。

**Comment by int64ago** [comment]
Updated with ONLY eslint modification @xuchaoying

File: package.json [hunk 3]
```diff
@@ -19,14 +19,12 @@
...
     "@commitlint/config-conventional": "^7.5.0",
+    "@svrx/eslint-config": "0.0.2",
```
[the remark this answers is not in the document]
Comment (xuchaoying) [reply package.json]: LGTM!

File: packages/svrx-util/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (xuchaoying) [reply packages/svrx-util/package.json]: outdated

File: packages/svrx/__tests__/spec/svrx.configure.js [hunk 4]
```diff
@@ -332,7 +332,7 @@ describe('Builtin Configs', () => {
     const server = createServer();
     const plugins = server.config.getPlugins();
     expect(plugins.length).to.eql(BUILTIN_PLUGIN.length);
-    expect(plugins.map(p => p.getInfo('name'))).to.eql(BUILTIN_PLUGIN);
+    expect(plugins.map((p) => p.getInfo('name'))).to.eql(BUILTIN_PLUGIN);
```
[the remark this answers is not in the document]
Comment (int64ago) [reply packages/svrx/__tests__/spec/svrx.configure.js]: It's the new behavior of airbnb eslint rules(maybe) with the latest version.

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (int64ago) [reply package.json]: Let's talk in English, it's a global project :-)

**Pull request merged**